samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -117,7 +136,31 @@ tests:
     build_only: true
     extra_configs:
       - CONFIG_LOCATION_SERVICE_EXTERNAL=y CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE=y
-    extra_args: SHIELD=nrf7002_ek OVERLAY_CONFIG="overlay-cloud_mqtt.conf;overlay-pgps.conf;overlay-nrf7002ek-wifi-scan-only.conf" DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    extra_args: SHIELD=nrf7002ek_nrf7002
+                OVERLAY_CONFIG="overlay-cloud_mqtt.conf;overlay-pgps.conf;overlay-nrf7002ek-wifi-scan-only.conf"
+                DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    tags: ci_build
+  sample.nrf9160.modem_shell.location_service_ext_pgps_nrf7000ek_wifi:
+    build_only: true
+    extra_configs:
+      - CONFIG_LOCATION_SERVICE_EXTERNAL=y CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE=y
+    extra_args: SHIELD=nrf7002ek_nrf7000
+                OVERLAY_CONFIG="overlay-cloud_mqtt.conf;overlay-pgps.conf;overlay-nrf7002ek-wifi-scan-only.conf"
+                DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay" CONFIG_WPA_SUPP=n
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    tags: ci_build
+  sample.nrf9160.modem_shell.location_service_ext_pgps_nrf7001ek_wifi:
+    build_only: true
+    extra_configs:
+      - CONFIG_LOCATION_SERVICE_EXTERNAL=y CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE=y
+    extra_args: SHIELD=nrf7002ek_nrf7001
+                OVERLAY_CONFIG="overlay-cloud_mqtt.conf;overlay-pgps.conf;overlay-nrf7002ek-wifi-scan-only.conf"
+                DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
```

Comment:
As suggested by @trantanen in https://github.com/nrfconnect/sdk-nrf/pull/10964#discussion_r1187179501, sample.nrf9160.modem_shell.location_service_ext_pgps_nrf7000ek_wifi and sample.nrf9160.modem_shell.location_service_ext_pgps_nrf7001ek_wifi could be removed. It should be enough to build the default configuration for all three EKs.

Reply:
Done